docs/publishers/config.rst

```diff
@@ -167,6 +187,22 @@ loads.
 
      This setting can only be set using :js:func:`window.hypothesisConfig`.
 
+   .. option:: onProfileRequest
+
+     ``function``. A JavaScript function that will be called when the user clicks
+     the user profile (user name) link in the sidebar. No arguments are passed
```

Comment:
If we ever want `@elife.org` users to be able to create annotations outside of the elife.org website, then having links on those annotations call a JavaScript callback function provided by elife.org won't work. It won't work if a user is viewing annotations on elife.org using their Chrome extension's client rather than the site's embedded one either. Or if `@elife.org` users annotations are shown on activity pages one day. Not saying we shouldn't do it just advising caution